test/signature.c

```diff
@@ -196,6 +278,8 @@ int main(int argc, char *argv[])
 	g_test_add_func("/signature/verify", signature_verify);
 	g_test_add_func("/signature/verify_file", signature_verify_file);
 	g_test_add_func("/signature/loopback", signature_loopback);
+	g_test_add_func("/signature/get_cert_chain", signature_get_cert_chain);
+	g_test_add_func("/signature/selfsigned", signature_selfsigned);
```

Comment:
The commit message states that we also check the intermediate CA case, which seems to be missing.

Reply:
Will have to remove this item from the commit message, as the commit re-introducing the 3-layer PKI comes afterwards. Intermediate cert tests for 2-layer PKI are useless.